Core/DISET/private/Transports/BaseTransport.py

```diff
@@ -36,7 +38,7 @@ def __init__( self, stServerAddress, bServerMode = False, **kwargs ):
     self.sentKeepAlives = 0
     self.waitingForKeepAlivePong = False
     self.__keepAliveLapse = 0
-    self.oSocket = None
+    self.oSocket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
```

Comment:
This specific line is probably not needed indeed, but without the proposed changes in PlainTransport there's no socket created at all. 
Which means that BTW as of now without this PR the "dip" protocol is not usable.